### DISCUSSION TITLE
Creation of pdfs too slow on Windows

### DISCUSSION THREAD

**Discussion by csgrig73** [pdfarranger/pdfarranger#1117] [General]
Has anyone else noticed an increase in output / save time in version 1.11.0?

I have version 1.10.0 installed on one PC (portable version) and saving times are normal. On two other PCs I installed version 1.11.0 (portable version on one PC, MSI installation on the other) and saving times are too slow. In fact, the more pages there are, the slower it takes to save the file.

Is this normal? What is the difference between 1.10.0 and 1.11.0 that lead to increased save times?

**Comment by kbengs** [comment]
#1110 also reported slow saving on 1.11.0 on Windows 10. There the speed was much faster when saving a second time. @csgrig73 do you see that behavior too? 

Here are the results of some tests i did:
Pdf with 3678 pages, open, then save:
Linux, 1.10.0, 3min0s
Linux, 1.11.0, 2min22s
Windows 10, 1.10.0, 6min30s
Windows 10, 1.11.0, 6min30s

I did a few test. The results are the average save time.

So for me the save time is same or better on 1.11.0. I guess it depends on the content in the PDF too, some might be faster on 1.10, other on 1.11. @csgrig73 do you want to share the PDF you tested with?

What save time do you see in 1.10.0 and 1.11.0 ?

**Reply by dreua** [reply]
In the old days when I still used Windows I often found antivirus deamons (even the defender) to be the cause of otherwise inexplicable slowdowns.

**Comment by dreua** [comment]
Closing this for lack of response on the one hand and as duplicate on the other. @csgrig73 your input could be helpful in #1110 so we can see if it all fits into one pattern and hopefully reproduce and fix it.

**Comment by dreua** [comment]
> Is this normal? What is the difference between 1.10.0 and 1.11.0 that lead to increased save times?

To answer this: It might be not in PDF Arranger but in an updated version of one of the included libraries. The PDF Arranger code is unlikely to perform different on Windows and Linux and we have no reports of this on Linux.
